lib/tictactoe.rb

```diff
@@ -1,52 +1,53 @@
 # frozen_string_literal: true
 
 class TicTacToe
-  def initialize(io:)
-    @board = Array.new(9, ' ')
-    @current_token = 'X'
+  def initialize(io:, board:, presenter:, move_validator:)
```

Comment:
It's a bit unclear to me what the responsibility of this class is. It seems to be handling everything from rendering to game logic to game loop.